cypress/support/commands/rancher-api-commands.ts

```diff
@@ -290,3 +290,20 @@ Cypress.Commands.add('setRancherResource', (prefix, resourceType, resourceId, bo
       expect(resp.status).to.eq(200);
     });
 });
+
+/**
+ * Get user preferences
+ */
+Cypress.Commands.add('getUserPreferences', () => {
+  return cy.request({
+    method:  'GET',
+    url:     `${ Cypress.env('api') }/v1/userpreferences`,
+    headers: {
+      'x-api-csrf': token.value,
+      Accept:       'application/json'
+    }
+  })
+    .then((resp) => {
```

Comment:
this isn't needed. though to match the pattern above it should have an `expect` in